src/pages/App.svelte

```diff
@@ -95,9 +95,7 @@
 <Layout>
   <div slot="sidebar">
     <div>
-      <button type="button" on:click={() => window.open("index.html")}>
-        Home</button
-      >
+      <button type="button" on:click={() => window.location.replace("/index.html")}> Home</button>
```

Comment:
```suggestion
      <button type="button" on:click={() => window.location.href = "index.html"}> Home</button>
```

The absolute path breaks on Github pages; try it from https://acteng.github.io/atip/home-without-new-tab/scheme.html?authority=Adur